WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -746,6 +747,10 @@ extension NotificationsViewController {
                         return
                     }
 
+                    self.notificationCommentDetailCoordinator.onSelectedNoteChange = { note in
+                        self.selectRow(for: note)
+                    }
```

Comment:
Perhaps we need a `[weak self]` here? The coordinator is strongly retained by `NotificationsViewController`, and `onSelectedNoteChange` closure is retained by the coordinator.

```suggestion
                    self.notificationCommentDetailCoordinator.onSelectedNoteChange = { [weak self] note in
                        self?.selectRow(for: note)
                    }
```